### DISCUSSION TITLE
2023-03-27 17点多，23号课程重新测试之后，成绩好像不会刷新了

### DISCUSSION THREAD

**Discussion by Dian-Kido** [AmazingAng/WTF-Solidity#461] [General]
2023-03-27 17点多，23号课程重新测试之后，成绩好像不会刷新了

**Comment by AmazingAng** [comment]
啥意思，这两天还有人成功通过，成功获得sbt

**Reply by Dian-Kido** [reply]
今天下午17点多开始的，早些时间都是好好的，做到第23课的时候，发现测试好几遍错误率不变了，然后我故意把题目选错，发现错误率还是没有变化

**Reply by AmazingAng** [reply]
看下置顶帖子

**Reply by Dian-Kido** [reply]
啊，了解了，感谢解答

**Reply by Dian-Kido** [reply]
> 啥意思，这两天还有人成功通过，成功获得sbt

<img width="1044" alt="image" src="https://user-images.githubusercontent.com/7953424/227931856-bf0ba600-d539-4490-a3cc-d1b9c7d4de08.png">

大佬，这题的选项，好像都少了一个")"
